tardis_env27.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
 - matplotlib=2.0
 - astropy=1.3
 - numexpr=2.6
-- Cython=0.28.4
+- Cython=0.29
```

Comment:
Let's test offline whether this version works. We could add a "troubleshooting" section to the docu about cython issues and our workaround, namely compiling cython by hand.